src/TurbulenceClosures/turbulence_closure_implementations/TKEBasedVerticalDiffusivities/time_step_catke_equation.jl

```diff
@@ -119,6 +119,7 @@ end
     #       Lᵂ = - Cᵂϵ * √e / Δz.
 
     on_bottom = !inactive_cell(i, j, k, grid) & inactive_cell(i, j, k-1, grid)
+    mask = !inactive_cell(i, j, k, grid)
```

Comment:
can we call this `not_inactive`?

I'm worried after a discussion with @milankl that we use the term `mask` inconsistently. A mask is 1 when applied, 0 when not. But here, masking it defined by `mask=false`. For the time being I think a good choice is to use more specific names, until we come up with a clean and consistent terminology for this